src/main/java/be/seeseemelk/mockbukkit/entity/PlayerMock.java

```diff
@@ -2613,15 +2615,13 @@ public boolean hasResourcePack()
 	@Override
 	public @NotNull PlayerProfile getPlayerProfile()
 	{
-		// TODO Auto-generated method stub
-		throw new UnimplementedOperationException();
+		return this.playerProfile;
 	}
 
 	@Override
 	public void setPlayerProfile(@NotNull PlayerProfile profile)
 	{
-		// TODO Auto-generated method stub
-		throw new UnimplementedOperationException();
+		this.playerProfile = profile;
```

Comment:
```suggestion
        Preconditions.checkNotNull(profile, "Profile cannot be null");
		this.playerProfile = profile;
```

And please add a Test for it

Reply:
Add to commit bbe7a424669c78c0fd91e520ec7848bc6a552502